Makefile.PL

```diff
@@ -2,11 +2,10 @@ use ExtUtils::MakeMaker;
 require "./pdlmaker.plm";
 pdlmaker_init();
 
-##-- preqrequistes (for PREREQ_PM)
+##-- prerequistes (for PREREQ_PM)
```

Comment:
replace typo with another typo? ;-)

Reply:
Ha ha! Let's call it partial replacement?